examples/resources/elasticstack_kibana_action_connector/resource.tf

```diff
@@ -10,3 +10,22 @@ resource "elasticstack_kibana_action_connector" "example" {
   })
   connector_type_id = ".index"
 }
+
+resource "elasticstack_kibana_action_connector" "pagerduty-connector" {
+  name = "pagerduty"
+  connector_type_id = ".pagerduty"
+  config = jsonencode({
+    apiurl = "https://events.pagerduty.com/v2/enqueue"
```

Comment:
```suggestion
    apiUrl = "https://events.pagerduty.com/v2/enqueue"
```

Reply:
Can you please also modify the [appropriate integration test](https://github.com/elastic/terraform-provider-elasticstack/blob/main/internal/kibana/connector_test.go#L457).